NRedisTimeSeries/TimeSeriesClient.cs

```diff
@@ -26,9 +26,9 @@ public static partial class TimeSeriesClient
         /// You can alter the default TSDB chunk size by passing the chunk_size argument (in Bytes)</param>
         /// <param name="policy">Optinal: Define handling of duplicate samples behavior (avalible for RedisTimeseries >= 1.4)</param>
         /// <returns>If the operation executed successfully</returns>
-        public static bool TimeSeriesCreate(this IDatabase db, string key, long? retentionTime = null, IReadOnlyCollection<TimeSeriesLabel> labels = null, bool? uncompressed = null, long? chunkSizeBytes = null, TsDuplicatePolicy? policy = null)
+        public static bool TimeSeriesCreate(this IDatabase db, string key, long? retentionTime = null, IReadOnlyCollection<TimeSeriesLabel> labels = null, bool? uncompressed = null, long? chunkSizeBytes = null, TsDuplicatePolicy? duplicatePolicy = null)
```

Comment:
please change the documentation of the functions after this rename
here and on the async api